### PR TITLE
add footer links and styles properly

### PR DESCRIPTION
**Pull request by kelvin8773**
* Add footer links
* styles up properly with zeplin design

### CODE REVIEW

File: src/footer/sign.svg:Zone.Identifier [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
```
Comment (jkettmann) [inline src/footer/sign.svg:Zone.Identifier]: What is this file doing? If it's not used please remove it

File: src/footer/footer.module.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    font-size: 14px;
+}
+
+.weblink {
+    margin-right: 397px;
+}
+
+.logolink {
+    margin-right: 341px;
+}
```
Comment (jkettmann) [inline src/footer/footer.module.scss]: Using fixed margins works for the exact size 1440px. But this prevents the app from being responsive. It's not an acceptance criterion in the ticket but you should keep responsiveness in mind for any project you're working on. 

There is another solution using `flex: 1` on the text links. This way the text links will take up the same amount of space while the logo just uses what's necessary.

File: src/footer/footer.module.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
+.footer {
+    height: 100px;
+    display: flex;
+    justify-content: flex-start;
```
Comment (jkettmann) [inline src/footer/footer.module.scss]: `justify-content: flex-start;` can be removed

Comment (kelvin8773) [reply src/footer/footer.module.scss]: ok, removed.

File: src/footer/sign.svg:Zone.Identifier (hunk 1 above)
Comment (kelvin8773) [reply src/footer/sign.svg:Zone.Identifier]: removed.

File: src/footer/footer.module.scss (hunk 2 above)
Comment (kelvin8773) [reply src/footer/footer.module.scss]: fair, it solved the problem.

**Comment by kelvin8773** [comment]
ok, fixed now!

**Pull request merged**